Example/ViewController.swift

```diff
@@ -472,7 +475,7 @@ class ViewController: UIViewController {
     }
 
     func navigationService(route: Route, routeIndex: Int, options: RouteOptions) -> NavigationService {
-        let mode: SimulationMode = simulationButton.isSelected ? .always : .onPoorGPS
+        let mode: SimulationMode = simulationButton.isSelected ? .always : .never
```

Comment:
The previous code here was correct: when the button was deselected, simulation was supposed to only kick in when the user enters a tunnel or otherwise loses GPS reception. The bug report in #2883 is that `always` has no effect: no simulation ever takes place when the option is enabled.